bkg/v2/BKG_v2.0.0.yaml

```diff
@@ -6718,9 +6642,9 @@ components:
             description: |
               The unique identifier for the equipment, which should follow the BIC ISO Container Identification Number where possible.
 
-              According to [ISO 6346](https://en.wikipedia.org/wiki/ISO_6346), a container identification code consists of a 4-letter prefix and a 7-digit number (composed of a 3-letter owner code, a category identifier, a serial number, and a check-digit).
+              According to [ISO 6346](https://www.iso.org/standard/83558.html), a container identification code consists of a 4-letter prefix and a 7-digit number (composed of a 3-letter owner code, a category identifier, a serial number, and a check-digit).
```

Comment:
This link is pointing to ISO 6346:2022, so is that the correct one?

Reply:
Flavia and Dominique requested us to point to this page and it was agreed that `:2022` should be removed from the visible part of the link